### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KutsenkoRN**

### CODE REVIEW

File: Logger.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿namespace H_W_Mod2_5
+{
+    public enum LogType
```
Comment (vladboss61) [inline Logger.cs]: separate to another file

File: FileService.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace H_W_Mod2_5
+{
+    internal interface IFileService
+    {
+        void Save(string messages);
+    }
+
+    internal class FileService : IFileService
```
Comment (vladboss61) [inline FileService.cs]: sealed

File: FileService.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using static System.Net.WebRequestMethods;
+using File = System.IO.File;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace H_W_Mod2_5
+{
+    internal interface IFileService
```
Comment (vladboss61) [inline FileService.cs]: separate to another file

File: BusinessException.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace H_W_Mod2_5
+{
+        internal class BusinessException : Exception
```
Comment (vladboss61) [inline BusinessException.cs:10]: add [Serialazable] attribute

File: Actions.cs [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+namespace H_W_Mod2_5
+{
+    internal interface IActions
+    {
+        Result LogInfoActionExample();
+        Result LogWarningActionExample();
+        Result LogErrorActionExample();
+    }
+
+    internal class Actions : IActions
```
Comment (vladboss61) [inline Actions.cs]: sealed

File: Actions.cs [hunk 6]
```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace H_W_Mod2_5
+{
+    internal interface IActions
```
Comment (vladboss61) [inline Actions.cs]: separate to another file

File: Logger.cs (hunk 1 above)
Comment (KutsenkoRN) [reply Logger.cs]: corrected

File: FileService.cs (hunk 2 above)
Comment (KutsenkoRN) [reply FileService.cs]: corrected

File: FileService.cs (hunk 3 above)
Comment (KutsenkoRN) [reply FileService.cs]: corrected

File: BusinessException.cs (hunk 4 above)
Comment (KutsenkoRN) [reply BusinessException.cs:10]: corrected

File: Actions.cs (hunk 5 above)
Comment (KutsenkoRN) [reply Actions.cs]: corrected

File: Actions.cs (hunk 6 above)
Comment (KutsenkoRN) [reply Actions.cs]: corrected

**Pull request merged**